template/codeland.rb

```diff
@@ -24,11 +24,16 @@ def rails_4?
   puts 'Please upgrade to ruby 2.2 or more'
 end
 
+gem_group :development do
+  gem 'rack-mini-profiler'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.